File: FrontEnd/src/app/admin/Models/InventoryModel.ts
import { Airline } from "./Airline";
import { AirlineModel } from "./AirlineModel";

export class InventoryModel{
    public constructor(
        public flightNumber:number,
        public airline:AirlineModel,
        public fromPlace:string,
        public toPlace:string,
        public startDate:any,
        public startTime:string,
        public endDate:any,
        public endTime:string,
        public schedule:string,
        public instrumentUsed:string,
        public total_business_class_seats:number,
        public total_economy_class_seats:number,
        public ticketPrice:number,
        public no_of_rows:number,
        public mealType:string,
    ){

    }
}

File: Backend/airline/src/main/java/com/airline/model/InventoryModel.java
package com.airline.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class InventoryModel {
	private long flightNumber;
	private String airline;
	private String fromPlace;
	private String toPlace;
	private String startDate;
	private String startTime;
	private String endDate;
	private String endTime;
	private String schedule;
	private String instrumentUsed;
	private int total_business_class_seats;
	private int total_economy_class_seats;
	private double ticketPrice;
	private int no_of_rows;
	private String mealType;
}
